File: frontend/src/app/dto/warehouse/UpdateWarehouseDTO.ts
export interface UpdateWarehouseDTO {
  name?: string;
  countryId?: number;
  cityId?: number;
  street?: string;
  streetNumber?: string;
  latitude?: number;
  longitude?: number;
  totalCapacity?: number;
  imagesToDelete?: number[];
  sectors?: UpdateWarehouseSectorDTO[];
}

export interface UpdateWarehouseSectorDTO {
  id?: number;
  name: string;
  minTemperature: number;
  maxTemperature: number;
  capacity: number;
}

File: backend/src/main/java/nvt/backend/dto/warehouse/UpdateWarehouseDTO.java
package nvt.backend.dto.warehouse;

import lombok.Data;

import java.util.List;

@Data
public class UpdateWarehouseDTO {
    private String name;
    private Long countryId;
    private Long cityId;
    private String street;
    private String streetNumber;
    private Double latitude;
    private Double longitude;
    private Double totalCapacity;
    private List<Long> imagesToDelete;
    private List<UpdateWarehouseSectorDTO> sectors;
}
